rfd/0036/README.md

```diff
@@ -67,23 +118,141 @@ These types may include:
+
+[Read more about what projects mean in the Mariposa context](./projects), including Triton CLI commands and the manifest file.
 
-This document defines tasks specific to deploying services, but the task queue is not limited to services.
 
 ### Project meta (including secrets)
 
 Many applications require configuration values which are undesirable or unsafe to set in the application image. These can include license keys, a flag setting whether it's a staging or production environment, usernames and passwords, and other details.
 
 This document proposes a simple method of storing those details and injecting them into containers. It is not intended to provide the rich features of solutions like Hashicorp's Vault, instead it is intended to provide a basic solution that is easy to use in a broad variety of applications.
+
+[Read more about meta in the Mariposa context](./meta), including Triton CLI commands and the manifest file.
+
+
+### Task queue
+
+Scaling, upgrading, even stopping all the instances of an service can take time...sometimes significant time. To represent this to the user, Triton must expose the task queue and offer the ability to cancel jobs.
+
+[Read more about the the Mariposa task queue](./queue), including Triton CLI commands.
+
+
+## User stories
+
+The following user stories are intended to provide a narrative understanding of how these features are intended to be used:
+
+- [jupiter.example.com: what it is and development workflow](./user-stories/jupiter-example-com.md)
+- [Automatically building and testing jupiter.example.com with Jenkins](./user-stories/jupiter-example-com-jenkins.md)
+- [Running jupiter.example.com in multiple data centers](./user-stories/jupiter-example-com-multi-dc.md)
+- [Health-checking, monitoring, and scaling jupiter.example.com](./user-stories/jupiter-example-com-monitoring-and-health.md)
+- [Creating, copying, and moving projects like microsite.jupiter.example.com](./user-stories/microsite-jupiter-example-com.md) (also includes secret management)
+
+
+## Implementation and architecture
+
+This RFD is mostly concerned with what users can do with Mariposa, not how those features are implemented. Work to define an implementation has begun in additional RFDs, which propose the following components:
+
+- [Projects API](https://github.com/joyent/rfd/blob/master/rfd/0079/README.md), which is where [projects](https://github.com/joyent/rfd/blob/master/rfd/0036/project.md) and their attached [services](https://github.com/joyent/rfd/blob/master/rfd/0036/service.md) and [metadata](https://github.com/joyent/rfd/blob/master/rfd/0036/meta.md) are managed
+- [ProjectsConvergence API](https://github.com/joyent/rfd/blob/master/rfd/0080/README.md), which is responsible for watching the actual state of the project and its services, comparing that to the goal state, developing a plan to reconcile those differences, and maintaining a queue of reconciliation plans currently being executed.
```

Comment:
I'm not sure I like `ProjectsConvergence` as the name here. `ProjectsWatcher` perhaps?